### ISSUE TITLE
Change T2star for T2starw as per new BIDS convention

### ISSUE THREAD

**Issue by jcohenadad** [sct-pipeline/spine-generic#50]
This change will need to be implemented once the BEP is merged on the official BIDS master branch. See the WIP proposal: https://github.com/bids-standard/bep001/blob/indexable_metadata/src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

**Comment by jcohenadad** [comment]
@agahkarakuzu Do you happen to know what is the status of this proposed changed?

**Comment by agahkarakuzu** [comment]
@jcohenadad you can find it [here](https://bep001.readthedocs.io/en/dev/04-modality-specific-files/01-magnetic-resonance-imaging-data.html#conventional-mri-suffixes).

**Comment by jcohenadad** [comment]
> @jcohenadad you can find it [here](https://bep001.readthedocs.io/en/dev/04-modality-specific-files/01-magnetic-resonance-imaging-data.html#conventional-mri-suffixes).
> 
> The repo is now at https://github.com/bids-bep001/bids-specification

cool 👍 so still a draft. But likely to become official at some point, right?

**Comment by agahkarakuzu** [comment]
Yes, but this should be close to the final version. We’ll open it up to discussion during ohbm. Fingers crossed 🤞🏻

**Comment by jcohenadad** [comment]
@agahkarakuzu do you have any update on this?

**Comment by agahkarakuzu** [comment]
@jcohenadad we are sending an email next week to the larger community, but my gut feeling is that `T2starw` would survive it :)
